deployment_scripts/deploy_windows.ps1

```diff
@@ -44,9 +71,12 @@ function Deploy-Windows([String] $monkey_home = (Get-Item -Path ".\").FullName,
     try
     {
         $version = cmd.exe /c '"python" --version  2>&1'
-        if ( $version -like 'Python 3.*' ) {
+        if ($version -like 'Python 3.*')
```

Comment:
If it's 3.6 we're f*cked and it won't work. Check if 3.7

Reply:
Are we incompatible with 3.6?

Reply:
I believe so.

Reply:
@ShayNehmad roping you in on this

Reply:
IIRC [Postponed Evaluation of Annotations](https://docs.python.org/3/whatsnew/3.7.html#whatsnew37-pep563) breaks it somewhere but I can't find where

Reply:
Won't change it for now. If someone shows up and says it breaks it, i'll change it.
Currently I have had parts of the monkey run in 3.5 and worked ok.